docker-compose.yml

```diff
@@ -20,24 +20,39 @@ services:
       - AWS_REGION
     ports:
       - "9090:9090"
-  app:
+  catalogs-aviris:
     build:
-      context: .
+      context: src/catalogs/aviris
+      dockerfile: Dockerfile
+    environment:
+      - AWS_PROFILE=${AWS_PROFILE:-nasa}
+    image: nasa-hsi-catalogs-aviris
+    volumes:
+      - ./src/catalogs/aviris:/usr/local/src:cached
+      - $HOME/.aws:/root/.aws:ro
+  activator-aviris-l2:
+    build:
+      context: src/pipeline/activator/aviris-l2
       dockerfile: Dockerfile
-      target: app
     environment:
+      - AAL2_FRANKLIN_URL=http://franklin:9090
+      - AAL2_S3_BUCKET=aviris-data-dev-${USER}
       - AWS_PROFILE=${AWS_PROFILE:-nasa}
-    image: nasa-hyperspectral_app
+    image: nasa-hsi-activator-aviris-l2
     volumes:
-      - .:/workspace:cached
+      - ./src/pipeline/activator/aviris-l2:/usr/local/src:cached
+      - pipeline-data:/data
       - $HOME/.aws:/root/.aws:ro
   dev:
     build:
       context: .
-      dockerfile: Dockerfile
+      dockerfile: Dockerfile.dev
       target: dev
```

Comment:
I ran into an issue with `scripts/update`:

```console
. . .
Building dev
ERROR: failed to reach build target dev in Dockerfile
```

```suggestion
```

Also, I'm following the directions in https://code.visualstudio.com/docs/remote/containers, but I'm unable to get the "Open folder in container" option to appear.

Reply:
Suggested change fixed the issue